tests/from.bats

```diff
@@ -427,8 +427,8 @@ load helpers
   run_buildah push $WITH_POLICY_JSON --tls-verify=false --creds testuser:testpassword --encryption-key jwe:${TEST_SCRATCH_DIR}/tmp/mykey.pub busybox oci:${TEST_SCRATCH_DIR}/tmp/busybox_enc
 
   # Try encrypted image without key should fail
-  run_buildah 125 from oci:${TEST_SCRATCH_DIR}/tmp/busybox_enc
-  expect_output --substring "decrypting layer .* missing private key needed for decryption"
+  run_buildah 1 from oci:${TEST_SCRATCH_DIR}/tmp/busybox_enc
```

Comment:
This exit code changes because the error stack, ultimately, contains an `exec.ExitError` from invoking an untar subprocess; and the top level of Buildah checks for `exec.ExitError`, and returns that exit code.

One can read that as a failure of the Go error wrapping design… or, alternatively, as Builder’s `main` matching too loosely if it wants only specific subprocesses’ exit codes to be returned unmodified.

AFAICS the exit code from Buildah is, apart from `buildah manifest exists`, not documented, so this PR just changes tests to accept the status 1. If we do actually commit (whether in documentation in or in practice) to returning 125 on unspecific failures, and this needs to be made more precise, please let me know (and also define which subprocess invocations that could fail with `ExitError` _should_ be propagated.)